package.json

```diff
@@ -31,27 +31,27 @@
         "commands": [
             {
                 "command": "markdown.extension.toc.create",
-                "title": "Markdown: Create Table of Contents"
+                "title": "%markdown.extension.toc.create.title%"
```

Comment:
I would like to use prefix `command` for these locale strings so we can easily know where they are used. And we can also remove `markdown.extension` for simplity.

I.e. `%command.toc.create.title%` here